apps/meteor/server/models/raw/Messages.ts

```diff
@@ -424,14 +429,25 @@ export class MessagesRaw extends BaseRaw<IMessage> implements IMessagesModel {
 		return this.find(query, options);
 	}
 
-	findLivechatMessagesWithoutClosing(rid: IRoom['_id'], options?: FindOptions<IMessage>): FindCursor<IMessage> {
-		return this.find(
-			{
-				rid,
-				t: { $exists: false },
-			},
-			options,
-		);
+	findLivechatMessagesWithoutTypes(
```

Comment:
Instead of creating a new func, can we reuse the `findVisibleByRoomIdNotContainingTypesBeforeTs` one?

Reply:
I was thinking the same thing, but both functions have a small difference related to getting the closing message and fetching the messages before the timestamp of closing message, so I wasn't sure.

From what I could find (and my limited knowledge on Omnichannel), there could still be system messages after closing the chat, so we might want to show these messages in case of PDF transcript? 

Correct me, if I'm wrong.

Reply:
There should be no system message after closing the chat. The only message sent _after_ closing is the message with the PDF itself 👀 so we should be safe doing the same the email does, of defaulting to `new Date()` to get the messages up to that point.

Reply:
Got it, will implement accordingly

Reply:
I checked this.

Just to clarify, I didn't create a new function to fetch the messages, rather updated the existing function used.

Also was checking why we were already using different functions, I stumbled upon a PR (by you) #28165 , I see why this was done using a different strategy on email transcripts, while we need all messages on the pdf transcripts.

I don't see any issues using different functions for both, or maybe we can merge both the functions as pass `beforeTs` as an optional param?

Reply:
Discussed over DM and decided to keep both functions as separate.